.github/workflows/checks.yml

```diff
@@ -15,6 +15,12 @@ jobs:
         ]
 
     runs-on: ${{ matrix.os }}
+    env:
+      ENABLE_INTEGRATION_TESTS: "true"
+      ENABLE_GAS_TESTS: "false"
```

Comment:
I was under the impression that:

1. we'll be not enabling integration tests at all on CI for now (as per offline discussion - there were problems with timeouts)
2. there was a discussion in here regarding enabling/disabling tests using _**single**_ env variable OR a flag to lib:test, and I stand by this idea. I don't think we will ever enable gas tests on CI, or any tests requiring fee for that matter. All the integration tests should be runnable easily locally using (ideally) one command, and that's it.

Reply:
1. Correct, but the reason was temporary outages of integration network which appear to be resolved as of now.

2. There was, but I was under the impression the idea was to get rid of 1+n variables. 
I believe we need at least those 3 states, which cannot be achieved with just one boolean:
1) disabling all integration tests 
2) enabling only non-gas tests
3) enabling both gas and non-gas tests

Keep in mind that AccountTest contains both tests that do require gas and those which do not (with the idea that eventually AccountTest, ProviderTest, etc. will mostly mimic the test suites for devnet).
So just passing `./gradlew :lib:test -x AccountTest` won't achieve the desired functionality.

Now, the idea is that those variables have the default values (which satisfies state 2).
Locally, expected commands are either of those:
`./gradlew :lib:test -PenableIntegrationTests=false` (1)
`./gradlew :lib:test -PenableGasTests=true` (3)

So we are still realistically passing one flag. If there's no flag, it's looking up those values in env. I think it would be nice fallback option for CI, where passing extra flags to `./gradlew :lib:koverXmlReport --info` would be less clear since the `:lib:test` is called indirectly.

We can of course tweak this to have both vars set to false by default, and we can make `PenableGasTest=true` override `PenableIntegrationTests=false` so we can be sure one flag is enough. If you insist on having one variable including defaults, we can consider something like:
`INTEGRATION_TEST_MODE` with possible values [`disabled`, `non_gas`, `all`] and explain it in readme. 

But I don't think we should complicate enabling any of those states, as even in matter of short time I found myself switching between those quite often.

Reply:
For now disabled integration tests on CI as per offline discussion and the fact that they are unstable today.

Reply:
ad 1. Temporary problems are troublesome, because they can easily come back. Even though integration works now, we should not enable it for CI, probably ever.
ad 2. We'll be only running those tests locally, from time to time. There should only be two scenarios: run integration tests or do not run integration tests. 

In order to push this PR forwards we can leave the discussion at this point, but please make sure you make an issue for this and change this behaviour.

Reply:
Discussed offline, tests will stay disabled, one variable with 3 possible values will be added.